odoo/orm/fields_textual.py

```diff
@@ -579,3 +579,8 @@ def convert_to_read(self, value, record, use_display_name=True):
     def get_trans_terms(self, value):
         # ensure the translation terms are stringified, otherwise we can break the PO file
         return list(map(str, super().get_trans_terms(value)))
+
+    escape = staticmethod(markup_escape)
```

Comment:
I don't think we should expose `escape`, devs should use `from markupsafe import Markup` object directly. `Markup.escape(my_string)`

Reply:
Hey @kmagusiak , thanks for your quick reviews. You're always one of the firsts to jump in 🙏🏻 

Alright. I've changed it 😉

What motivated me to include it here is that, in the codebase, pretty much all cases are doing this:

```py
from odoo.tools import html_escape

value = html_escape(sometext)
```

and `fields.Html.escape(sometext)` seems more idiomatic.

Reply:
Ok], it can stay as-is. I think these are usages introduced before `markupsafe` was used.
@rco-odoo what do you think?

Reply:
Alright :) it's back on ^^
Thanks!